cli/command/manifest/fetch.go

```diff
@@ -44,23 +43,21 @@ func getLocalImageManifestData(namedRef reference.Named, transactionID string) (
 		return nil, nil, err
 	}
 
-	// If this is a manifest list, let's check for it locally so a user can see any modifications
-	// he/she has made.
+	// TODO: is this case relevant if transactionID == "" ?
```

Comment:
Sure, if it's easier to review squashed I can do that. I'll close #3 since it's incldued here and squash this into a single commit. I can squash everything in #5 as well